packages/react-components/src/select/src/Select.tsx

```diff
@@ -1,115 +1,132 @@
 import "./Select.css";
 
+import { CSSProperties, ComponentProps, ElementType, ForwardedRef, ReactElement, ReactNode, SyntheticEvent } from "react";
 import { DisclosureArrow } from "../../disclosure";
 import { HiddenSelect } from "./HiddenSelect";
+import { InteractionStatesProps, augmentElement, cssModule, forwardRef, mergeProps } from "../../shared";
 import { Listbox } from "../../listbox";
 import { Overlay } from "../../overlay";
 import { Text } from "../../text";
-import { any, bool, element, elementType, func, number, object, oneOf, oneOfType, string } from "prop-types";
-import { augmentElement, cssModule, mergeProps } from "../../shared";
-import { forwardRef } from "react";
 import { isNil } from "lodash";
 import { useFieldInputProps } from "../../field";
 import { useSelect } from "./useSelect";
 
-const propTypes = {
+export interface SelectMenuProps {
+    id?: string,
+    style?: CSSProperties,
+    [x: string]: any
+}
+
+export interface InnerSelectProps extends InteractionStatesProps {
+    /**
+     * @ignore
+     */
+    name?: string;
```

Comment:
Should it go in DomProps interface? Not sure what are the boundaries of this interface.

I guess not since "name" is only for "submitable" elements. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes